.env.example

```
@@ -0,0 +1,5 @@
+# API url
+VUE_APP_API_URL=ws://localhost:8080/client
```

```suggestion
VUE_APP_API_URL=ws://localhost:3000/client
```
Чтобы совпадало с .env.example в api.